pkg/apis/nais.io/v1alpha1/application_types.go

```diff
@@ -271,11 +288,23 @@ func (in *Application) LastSyncedHash() string {
 func (in *Application) SetLastSyncedHash(hash string) {
 	in.NilFix()
 	in.Annotations[LastSyncedHashAnnotation] = hash
+	in.Status.LastSyncedHash = hash
 }
 
 func (in *Application) SetCorrelationID(id string) {
 	in.NilFix()
 	in.Annotations[CorrelationIDAnnotation] = id
+	in.Status.CorrelationID = id
+}
+
+func (in *Application) SetDeploymentRolloutStatus(i int32) {
```

Comment:
* Two identical functions?
* An `int32` doesn't say anything here. It should be typed correctly with the `event.RolloutStatus` type.